dusty/payload.py

```diff
@@ -3,6 +3,14 @@
 
 from .constants import VERSION
 
+def _encode_utf8(args):
```

Comment:
I'm concerned about having to traverse a data structure like this.  We should talk about it